test/unit/org/apache/cassandra/index/internal/CassandraIndexTest.java

```diff
@@ -561,6 +562,20 @@ public void indexCorrectlyMarkedAsBuildAndRemoved() throws Throwable
         String selectBuiltIndexesQuery = String.format("SELECT * FROM %s.\"%s\"",
                                                        SchemaConstants.SYSTEM_KEYSPACE_NAME,
                                                        SystemKeyspace.BUILT_INDEXES);
+
+        // Wait for any background index clearing tasks to complete
```

Comment:
Maybe we could add a note about parallel runners, in case they come back in the future?
```suggestion
        // Wait for any background index clearing tasks to complete,
        // knowing that there aren't any other tests running parallely
```

Reply:
I'm sort of -1. Junit test methods _by definition_ should be 100% isolated. Junit itself goes to great lenghts to make sure each test case is isolated as much as possible. CASSANDRA-16595 is actually solving _a bug_ where we broke junit rules. I can add for completeness but I'll reword it.